src/main/java/jenkins/plugins/rpmsign/RpmSignPlugin.java

```diff
@@ -30,68 +36,70 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.StringTokenizer;
+import javax.annotation.Nonnull;
 
-public class RpmSignPlugin extends Recorder {
+public class RpmSignPlugin extends Recorder implements SimpleBuildStep {
 
-  private List<Rpm> entries = Collections.emptyList();
+  private List<Rpm> rpms;
 
   @DataBoundConstructor
   @SuppressWarnings("unused")
   public RpmSignPlugin(List<Rpm> rpms) {
-    this.entries = rpms;
-    if (this.entries == null) {
-      this.entries = Collections.emptyList();
+    this.rpms = rpms;
+    if (this.rpms == null) {
+      this.rpms = Collections.emptyList();
     }
   }
 
+  @DataBoundSetter
+  public void setRpms(final List<Rpm> rpms) {
+    this.rpms = rpms;
```

Comment:
Shouldn't this also have the null check like in the constructor?

Reply:
addressed